app/res/values/dimens.xml

```diff
@@ -26,6 +26,12 @@
     <dimen name="min_button_height">52dp</dimen>
     <dimen name="spacer">10dp</dimen>
 
+    <dimen name="menu_icon_size">60dp</dimen>
+    <dimen name="menu_grid_icon_size">120dp</dimen>
+    <dimen name="menu_grid_icon_size_with_badge">136dp</dimen>
+    <dimen name="numeric_badge_font_size">15dp</dimen>
+    <dimen name="numeric_badge_width">30dp</dimen>

```

Comment:
Hm, agree that they should either both be SP or both be DP. It is definitely the case that our _icons_ wont' scale with sp, so makes a lot of sense to me to frame/anchor the size of these components to the icon rather than the menu text. Sounds good!